doc/deprecation_process.md

```diff
@@ -2,57 +2,222 @@ This page can be linked using: [aka.ms/azsdk/python/deprecation-process](https:/
+ - the old package is scheduled to be deprecated on a specific date
+ - guidance on adjusting code to use the new package.
 
-   > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
-   >
-   > For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
+Reach out to the Python Azure SDK PM, Rohit Ganguly (rohitganguly), if you have any questions about creating a blog post.
 
-- `CHANGELOG.MD` add a new version with the current date, and the same disclaimer. For instance
+# Step 1: Updates to the package files
 
-   > ## 1.2.3.post1 (2023-03-31)
-   >
-   > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
-   >
-   > For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
-   
-   or
+Clone the `azure-sdk-for-python` repository and update the following files of your package.
 
-   > ## 1.2.4 (2023-03-31)
-   >
-   > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
-   >
-   > For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
+## README.md
 
-- `azure/mypackage/_version.py` : Change the version to the one used in the changelog (for instance `"1.2.3.post1"` or `"1.2.4"`). This file may be called `version.py` if your package is very old.
-- `sdk_packaging.toml` : You need to add `auto_update = false` if not already present to avoid the bot overriding your changes
-- `setup.py` change the `Development Status` classifier to `Development Status :: 7 - Inactive`. **Important: This needs to be your LAST commit on the PR. More on this at the bottom of this page if you want details.**.
+A disclaimer should be added indicating end-of-life date (EOLDate) of the package and directing to a replacement package and migration guide as necessary.
+  - The EOLDate should be in the format `MM-DD-YYYY`.
+    - If there is no replacement package, the package EOLDate should be the service retirement date. This does not necessarily need to be the same as the release date in the CHANGELOG.md, since the package may be deprecated before/after the service is retired.
+    - If there is a replacement package (or repo), the EOLDate should be the same as the deprecation release date of the old package in the CHANGELOG.md.
+    - Service retirement dates MAY be listed [here](https://aka.ms/servicesretirementworkbook), where retiring feature says 'Entire service'.
+  - The link to the replacement package should be a PyPI link: `https://pypi.org/project/azure-mynewpackage/`.
+  - The link to the migration guide should be an `aka.ms` link in the format `https://aka.ms/azsdk/python/migrate/my-new-package`. To create this link, go to [https://aka.ms/](https://aka.ms/).
```

Comment:
We have a tutorial page on how to create aka.ms that looks consistent from a package to another. @weshaggard would have the link to it. I'm afraid if we just tell people to straight go to aka.ms, link would look inconsistent. At the very least, we should suggest a syntax like aka.ms/azsdk/deprecation/packagename or something like that

Reply:
This is the syntax that we've used so far: https://aka.ms/azsdk/python/migrate/my-new-package`

But I'll try to find the link to the tutorial.

Reply:
updated!